### PR TITLE
Features/exclude headings

### PR DESCRIPTION
**Pull request by naokazuterada**
@jonasbn 

An update on the "Exclude heading" is now ready.
Could you please do a final check of [this section](https://github.com/naokazuterada/MarkdownTOC/blob/features/discrete-headings/README.md#excluded-heading) describing it?

Even if you can approve it, please do **not** merge it, because I will do the final release work.

Thanks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,10 +41,12 @@ Sublime Text 3 plugin for generating a Table of Contents (TOC) in a Markdown doc
     - [Customizable list bullets in TOC](#customizable-list-bullets-in-toc)
     - [Specify custom indentation prefix](#specify-custom-indentation-prefix)
     - [Preserve images in headings](#preserve-images-in-headings)
+    - [Excluded heading](#excluded-heading)
```
Comment (jonasbn) [inline README.md:44]: Rewrite to "Excluded headings", since we are describing the feature in generically

File: README.md [hunk 2]
```diff
@@ -98,6 +100,7 @@ The **MarkdownTOC** plugin is rich on features and customization, useful for bot
 - [Customizable list bullets in TOC](#customizable-list-bullets-in-toc)
 - [Specify custom indentation prefix](#specify-custom-indentation-prefix)
 - [Preserve images in headings](#preserve-images-in-headings)
+- [Excluded heading](#excluded-heading)
```
Comment (jonasbn) [inline README.md:103]: Rewrite to "Excluded headings", since we are describing the feature in generically

File: README.md [hunk 3]
```diff
@@ -722,6 +725,15 @@ Please note that the default for the [attribute](#attributes) is: `false`.
 
 You can change your default setting in your [configuration](#configuration) with the key `defaults.remove_image`.
 
+### Excluded heading
```
Comment (jonasbn) [inline README.md:728]: Rewrite to "Excluded headings", since we are describing the feature in generically

File: README.md [hunk 4]
```diff
@@ -722,6 +725,15 @@ Please note that the default for the [attribute](#attributes) is: `false`.
...
+
+You can exclude certain headings in the TOC by adding a special comment to the line before the heading, as shown below.
```
Comment (jonasbn) [inline README.md:730]: Suggestion: "You can exclude certain headings in the TOC by adding a special comment to the line above the line with the heading, as shown below."

**Comment by naokazuterada** [comment]
@jonasbn Thank you!! I've updated.

**Pull request merged**